packages/apollo-collaboration-server/src/export/export.controller.ts

```diff
@@ -41,11 +41,20 @@ export class ExportController {
   @Validations(Role.None)
   @Get()
   async exportGFF3(
-    @Query() request: { exportID: string; fastaWidth?: number },
+    @Query()
+    request: {
+      exportID: string
+      withFasta: 'True' | 'False'
+      fastaWidth?: number
+    },
```

Comment:
We should use pipes to parse and provide defaults for the query parameters: https://docs.nestjs.com/pipes

```suggestion
    @Query('exportID') exportID: string,
    @Query('includeFASTA', new DefaultValuePipe(false), ParseBoolPipe)
    includeFasta: boolean,
    @Query('fastaWidth', new DefaultValuePipe(80), ParseIntPipe)
    fastaWidth: number,
```